doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/internal/api/AccountClient.kt

```diff
@@ -29,8 +30,11 @@ internal open class AccountClient(
      */
     @DoordeckOnly
     suspend fun refreshTokenRequest(refreshToken: String): TokenResponse {
-        return httpClient.post(Paths.getRefreshTokenPath()) {
+        return httpClient.post<TokenResponse>(Paths.getRefreshTokenPath()) {
             addRequestHeaders(token = refreshToken)
+        }.also {
+            contextManager.setAuthToken(it.authToken)
+            contextManager.setRefreshToken(it.refreshToken)
```

Comment:
From now on, we will update the context values whenever possible. Previously, this had to be done manually by the developer. Please note that this only updates the variable; it does not persist it.